src/quo2/screens/permission_tag.cljs

```diff
@@ -0,0 +1,173 @@
+(ns quo2.screens.permission-tag
+  (:require [quo.react-native :as rn]
+            [quo.previews.preview :as preview]
+            [quo2.components.permission-tag :as permission-tag]
+            [quo2.foundations.colors :as colors]
+            [status-im.react-native.resources :as resources]
+            [reagent.core :as reagent]))
+
+(def descriptor [{:label   "Size:"
+                  :key     :size
+                  :type    :select
+                  :options [{:key   32
+                             :value "32"}
+                            {:key   24
+                             :value "24"}]}
+                 {:label "Locked:"
+                  :key   :locked
+                  :type  :boolean}])
+
+(def community-tokens
+  [{:token-groups   [{:id  1 :tokens [{:id 1 :token-icon (resources/get-image :status-logo)}]}]}
```

Comment:
you can reduce the size of this section in many ways. Consider at least using a small helper function to create the `tokens`

e.g

`(defn get-token-icons [length]
 (map (fn [id] 
  {:id id :token-icon (resources/get-image :status-logo)})
 (range length)))

(def community-tokens
   
  [
    {:token-groups   [{:id  10 :tokens (get-token-icons 1)}]}
   {:token-groups   [{:id  11 :tokens  (get-token-icons 2)}]}
   {:token-groups   [{:id  12 :tokens (get-token-icons 3)}]}


   {:token-groups   [{:id  13 :tokens (get-token-icons 4)}]}

   {:token-groups   [{:id  14 :tokens (get-token-icons 1)}
                     {:id  20 :tokens (get-token-icons 1)}]}

   {:token-groups   [{:id  103 :tokens (get-token-icons 1)}
                     {:id  203 :tokens (get-token-icons 2)}]}

   {:token-groups   [{:id  10 :tokens (get-token-icons 1)}
                     {:id  200 :tokens (get-token-icons 3)}]}`